src/main/webapp/app/entities/submission.model.ts

```diff
@@ -29,9 +29,16 @@ export abstract class Submission implements BaseEntity {
     public submissionExerciseType?: SubmissionExerciseType;
     public durationInMinutes?: number;
 
+    // results is initialized by the value the server returns
     public results?: Result[];
+
     public participation?: Participation;
 
+    // Helper Attributes
+
+    // latestResult is undefined until submission.setLatestResult() is called
```

Comment:
its called setLatestSubmissionResult I think

Reply:
Yea, I changed the comment